src/main/com/fulcrologic/guardrails/malli/core.cljc

```diff
@@ -86,7 +86,9 @@
            mode (gr.cfg/mode cfg)]
        ;; NOTE: Possibly manual override to always include them?
        (when (and cfg (#{:runtime :all} mode))
-         `(register! ~k ~v)))))
+         (if (symbol? k)
+           `(do (def ~k ~v) @gr.reg/schema-atom)
```

Comment:
I do not understand this code. Why would you dereference the schema atom???

Reply:
It is only to make both branches of the `if` return the same thing. The other branch returns schema atom as well. I know the actual output of this def operation is usually not used.